### DISCUSSION TITLE
Welcome to vnpy_crypto Discussions!

### DISCUSSION THREAD

**Discussion by vn-crypto** [veighna-global/vnpy_evo#1] [Announcements]
Hi, welcome to the vnpy_crypto discussions!

Since the official maintainer of [VeighNa](https://www.github.com/vnpy/vnpy) project no longer supports crypto trading, feel free to use this place for related topics!

**Comment by qinxi1122** [comment]
![Snipaste_2022-12-11_16-38-40](https://user-images.githubusercontent.com/15192895/206894209-129321b9-4a16-4825-b8ad-8004a0e18375.png)

**Reply by qinxi1122** [reply]
这个报错是怎么回事
